### PR TITLE
Fix hanoi source tags for device-virtual-go and app-service-config

### PR DESCRIPTION
**Pull request by siggiskulason**
Update the source tags for device-virtual-go and app-service-config

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -764,8 +764,8 @@ parts:
 
   app-service-config:
     source: https://github.com/edgexfoundry/app-service-configurable.git
-    # TODO(Hanoi): update to source-tag for release
-    source-branch: master
+    source-depth: 1
+    source-tag: v1.3.0
```
Comment (tonyespy) [inline snap/snapcraft.yaml:768]: why do you echo "1.3.0" to VERSION for device-virtual but no app-service-configurable?

**Review by tonyespy — COMMENTED** [review]
one comment inline...

File: snap/snapcraft.yaml (hunk 1 above)
Comment (siggiskulason) [reply snap/snapcraft.yaml:768]: Good point.There was no echo of VERSION there in place for app-service-configurable and I didn't add it in this PR. It should probably be in there as well, come to think of it. The old snapcraft.yaml file for app-service-configurable had a hardcoded version string so it didn't need the VERSION file. We have already updated that to use VERSION, so although a VERSION file was not required in the past we should really have one there now. I have updated this and pushed the updated commit.

**Review by tonyespy — APPROVED** [review]
LGTM

**Pull request merged**